app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -12,7 +12,7 @@ CONFIG_COMP_MULTIBAND_DRC=y
 CONFIG_PM=y
 CONFIG_PM_DEVICE=y
 CONFIG_PM_DEVICE_RUNTIME=y
-CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=n
+CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=y
```

Comment:
Some explanation about this option would be beneficial (the one-liner from Zephyr Kconfig should be ok). Also to understand this patch, it helps to understand this is set by default and this was disabled for specific Intel platforms only.